src/server.ts

```diff
@@ -2,9 +2,14 @@ import dotenv from 'dotenv';
 dotenv.config();
 
 import express from 'express';
+import { connectToMongoDb } from './lib/db';
 import router from './lib/router';
 
-const { PORT = 3001 } = process.env;
+const { PORT = 3001, MONGODB_URI } = process.env;
```

Comment:
could we get rid of it, in favour of direct imported env variables, after changing samples.http?